src/hooks/useInstantEffect.ts

```diff
@@ -3,7 +3,7 @@
  * For licensing, see LICENSE.md.
```

Comment:
I found a problem with our es rule - it's probably not directly related to the PR, but it's worth noting. Every file complains that the rule about the license key cannot be found: `Definition for rule 'ckeditor5-rules/prevent-license-key-leak' was not found.`

Reply:
fyi @filipsobol